packages/lodestar/src/network/util.ts

```diff
@@ -50,3 +55,105 @@ export async function initializePeerInfo(peerId: PeerId, multiaddrs: string[]):
   multiaddrs.forEach((ma) => peerInfo.multiaddrs.add(ma));
   return peerInfo;
 }
+
+export function encodeChunkifyResponse(config: IBeaconConfig, method: Method, {err, output}: ChunkResponse): Buffer {
```

Comment:
Can we call these functions `encodeResponseChunk` and `decodeResponseChunk` to match the `ResponseChunk` name?